packages/vscode-dependencies-validation/package.json

```diff
@@ -20,13 +20,18 @@
     "ci": "npm-run-all clean compile bundle",
     "clean": "rimraf ./dist *.vsix",
     "compile": "tsc -p ./",
+    "watch": "tsc -p ./ --watch",
```

Comment:
You are not using the top level watch?
someone could get confused and execute both the top level and sub-package levels watches...
is it worth the possible problem instead of just running the root level watch?

Reply:
Removed